### PR TITLE
enh(themes) GitHub v11

### PR DESCRIPTION
**Pull request by Hirse**
### Changes
- Update GitHub theme for v11
- Remove `github-gist` as it's outdated and superseded by `github`
- Add `github-dark` and `github-dark-dimmed` as github.com added them

When updating the GitHub theme with the new scopes I was once again struggling with how to style class and function names.
The actual highlighting on GitHub is very inconsistent cross languages:
GitHub highlighting
*Highlighting as image to snapshot current state, code is below*

In order not to overwrite the colors for each language, I tried to take a compromise with the best look:
C#
csharp hljs with github theme

Java
java hljs with github theme

JavaScript
js hljs with github theme

### Checklist
- [x] ~Added markup tests~, or they don't apply here because theme changes only
- [x] Updated the changelog at `CHANGES.md`

---

C#
```csharp
class Child : Parent {
  public Child() : base() {
    this.prop = "";
  }
  void getProp() {}
}
```

Java
```java
class Child extends Parent {
  public Child() {
    super();
    this.prop = "";
  }
  void getProp() {}
}
```

JavaScript
```js
class Child extends Parent {
  constructor() {
    super();
    this.prop = "";
  }
  getProp() {}
}
```

### CODE REVIEW

**Comment by joshgoebel** [comment]
> The actual highlighting on GitHub is very inconsistent cross languages:

I'd give up on any kind of exact match... it seems GitHub classifies things differently, so that's impossible.  For example `Parent` above should always be `title.class.inherited` for all grammars.

> with how to style class and function names.

If you have a specific question (other than the general frustration), let's go over it here - now that I've put in the work on a lot of other grammars and the new scope stuff and it's fresh.  But in general:

- `title.class`
- `title.class.inherited`
- `title.function`

And then the themes decide what to do with that.

File: CHANGES.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Themes:
 - The default padding of all themes increases (0.5em => 1em).
 - `schoolbook` has been updated to remove the lined background.
 - `github` updated to better match modern GitHub (#1616) [Jan Pilzer][]
+- `github-gist` has been removed in favor of `github` [Jan Pilzer][]
```
Comment (joshgoebel) [inline CHANGES.md:35]: This needs mention in VERSION_11.md as well.

File: CHANGES.md [hunk 2]
```diff
@@ -116,6 +117,7 @@ Theme Improvements:
 New Themes:
 
 - DeviBeans Dark by [Farzad Sadeghi][]
+- GitHub Dark and GitHub Dark Dimmed [Jan Pilzer][]
```
Comment (joshgoebel) [inline CHANGES.md:120]: Yes, please!!!!

File: CHANGES.md [hunk 3]
```diff
@@ -209,7 +212,6 @@ Deprecations:
 [David Ostrovsky]: https://github.com/davido
 [AndyKIron]: https://github.com/AndyKIron
 [Samuel Colvin]: https://github.com/samuelcolvin
-[Jan Pilzer]: https://github.com/Hirse
```
Comment (joshgoebel) [inline CHANGES.md:212]: Why?

**Comment by joshgoebel** [comment]
Looks good at a glance.

File: CHANGES.md (hunk 3 above)
Comment (Hirse) [reply CHANGES.md:212]: I didn't have a contribution in 10.7, but did have one in 10.6 so just moved this line down one block.

Comment (joshgoebel) [reply CHANGES.md:212]: Ah. ok :) Oops.

File: CHANGES.md (hunk 1 above)
Comment (Hirse) [reply CHANGES.md:35]: Added.

**Comment by Hirse** [comment]
Thanks for the summary, but I just meant to say "with how to style class and function names [for this theme]" as applying GitHub's style correctly seems almost impossible.

**Review by joshgoebel — APPROVED** [review]

**Pull request merged**